configs/openmrs_config/ampathforms/NCD_Screening.json

```diff
@@ -1076,10 +1076,19 @@
                                     {
                                         "concept": "9b305de5-9df6-4db9-820d-9a24541418ab"
                                     }
-                                ]
+                                ],
+                            "calculate": {
+                                "calculateExpression": "CVDscore < 20 ? '1dbd741d-b9d9-49d1-b021-90931a6bdd1a' : CVDscore >= 20 && CVDscore < 30 ?  'a085b404-2596-48d6-a454-cd164d4a83fc' : CVDscore >= 30 ? '9b305de5-9df6-4db9-820d-9a24541418ab' : null "
+                            }
                             },
                             "type": "obs",
-                            "validators": []
+                            "validators": [
+                                {
+                                    "type": "js_expression", 
+                                    "failsWhenExpression": "isEmpty(CVDscore) || isEmpty(CVDfollowUpScreening) || (CVDfollowUpScreening !== '1dbd741d-b9d9-49d1-b021-90931a6bdd1a' || CVDfollowUpScreening !== 'a085b404-2596-48d6-a454-cd164d4a83fc'|| CVDfollowUpScreening !== '9b305de5-9df6-4db9-820d-9a24541418ab')",                            
+                                    "message": "Please ensure that CVD Risk Score is calculated"
```

Comment:
@kazlaw when is this error expected to be thrown?

Reply:
I deally when the conditions set in `failsWhenExpression` are met

Reply:
What am observing is that when CVD score is empty, the **Follow-up screening CVD risk factors** in the Management section defaults to `Monitor every year (<20%)` meaning it considers the score to be less than 20%. In other words the error is not thrown which I think should be the case since Management sections contains other obs.

Reply:
@kazlaw let me know if we intend to have the error thrown on the UI when CVD risk score is empty?

Reply:
> What am observing is that when CVD risk score is empty, the **Follow-up screening CVD risk factors** in the Management section defaults to `Monitor every year (<20%)` meaning it considers the score to be less than 20%. In other words the error is not thrown which I think should be the case since Management section contains other obs.

I have modified the condition to cater for this. Good eye!